### PR TITLE
Avoid usage of new guava APIs

### PR DESCRIPTION
**Pull request by szarnekow**

### CODE REVIEW

File: org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/server/UriExtensionsTest.xtend [hunk 1]
```diff
@@ -270,7 +270,7 @@ class UriExtensionsTest {
 		var directory = createTempDir('some name with whitespaces').toFile;
 		assertTrue(directory.exists)
 		assertTrue(directory.directory)
-		assertTrue(CharMatcher.whitespace.matchesAnyOf(directory.name));
+		assertTrue(directory.name.contains(' '));
```
Comment (cdietrich) [inline org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/server/UriExtensionsTest.xtend]: why change this?

Comment (szarnekow) [reply org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/server/UriExtensionsTest.xtend]: My TP did not contain a Guava version with that method after I re-oomphed.

Comment (cdietrich) [reply org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/server/UriExtensionsTest.xtend]: test should be gradle oomphed and that one should use a newer guava

Comment (szarnekow) [reply org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/server/UriExtensionsTest.xtend]: So this is to be rejected?

**Comment by cdietrich** [comment]
Since it is a test I don’t care as long thing thing what should be tested is still tested

**Pull request closed without merging**